src/MeshKernelNET/Native/Mesh1DNative.cs

```diff
@@ -27,9 +27,19 @@ public struct Mesh1DNative
         /// </summary>
         public int num_nodes;
 
+        /// <summary>
+        /// The number of 1d valid nodes
+        /// </summary>
+        public int num_valid_nodes;
+
         /// <summary>
         /// The number of 1d edges
         /// </summary>
         public int num_edges;
+
```

Comment:
the number of valid edges can be computed, there is no need to make it a member.